pkg/volume/util/metrics.go

```diff
@@ -47,16 +47,7 @@ var storageOperationMetric = metrics.NewHistogramVec(
 		Buckets:        []float64{.1, .25, .5, 1, 2.5, 5, 10, 15, 25, 50, 120, 300, 600},
 		StabilityLevel: metrics.ALPHA,
 	},
-	[]string{"volume_plugin", "operation_name"},
-)
-
-var storageOperationErrorMetric = metrics.NewCounterVec(
-	&metrics.CounterOpts{
-		Name:           "storage_operation_errors_total",
-		Help:           "Storage operation errors",
-		StabilityLevel: metrics.ALPHA,
-	},
-	[]string{"volume_plugin", "operation_name"},
+	[]string{"volume_plugin", "operation_name", "status"},
```

Comment:
It seems that storageOperationEndToEndLatencyMetric suffers from the same problem, where errors don't have an e2e latency (see [RecordMetrics](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/persistentvolume/metrics/metrics.go#L292)).

But that seems to be a different sort of beast so rolling errors into that metric may not be desirable?

Reply:
yeah I think we need to revisit operationendtoend in general (not in this pr). The implementation is complex, and may be better suited to be tracked by an outside observer instead.

Reply:
so we can merge `volumeOperationErrorsMetric` to `storageOperationEndToEndLatencyMetric`. And as same as this MR, 
add a status to `storageOperationEndToEndLatencyMetric` ?

@mattcary
@msau42

Reply:
No, the end to end metric is much more complicated and I think needs a bit of redesign. I think we should just leave it as it is for now.